Sources/DreamsViewController.swift

```diff
@@ -25,18 +25,22 @@ public class DreamsView: UIView {
     internal var webService: WebServiceType = WebService()
     internal var dreams = Dreams.shared
 
-    public override init(frame: CGRect) {
-        super.init(frame: frame)
+    public override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+        super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
         setup()
     }
 
     public required init?(coder: NSCoder) {
```

Comment:
Is ability to instantiate from IB/SB a part of our requirements?

Reply:
Yes 🙂

Reply:
Ok got it, thank you. Then in this single case dependency injection will be impossible for that scenario but for all others it should be doable.